### ISSUE TITLE
GitUp crashes on git rebase

### ISSUE THREAD

**Issue by iovuio** [git-up/GitUp#296]
This happens repeatedly, however not every time. 
Once it will occur to me what the steps are to do this deterministically, I will update this issue.
Please see the macOS crash report contents in the file attached.
[iterm2-crash.log.zip](https://github.com/git-up/GitUp/files/879554/iterm2-crash.log.zip)

**Comment by daflair** [comment]
I'm still new to GitUp and just checking if there are issues to watch out for... did you run into the same problem again?

**Comment by iovuio** [comment]
Yes, all but the trivial rebases (run from the command line) end up in the
app crashing.

That is no problem, though. Start gitup anew and watch it shine!

Pe 26 apr. 2017 16:33, "Saleh Abdel Motaal" <notifications@github.com> a
scris:

**Comment by daflair** [comment]
Great... thanks!
